sdk/src/main/kotlin/io/rover/sdk/services/EventEmitter.kt

```diff
@@ -34,10 +34,26 @@ open class EventEmitter(
             action
         )
 
+        val analyticsName = EventAction.values().find { it.action == action }?.analyticsName
+
+        analyticsName?.let {
+            eventSubject.onNext(Event(it, attributes))
+        }
+
         intent.putExtra("attributes", JSONObject(attributes).toString())
 
         localBroadcastManager.sendBroadcast(intent)
         log.v("Event broadcast: $action, ${JSONObject(attributes).toString(4)}")
     }
 }
 
+enum class EventAction(val action: String, val analyticsName: String) {
+    EXPERIENCE_PRESENTED("io.rover.ExperiencePresented", "Experience Presented"),
+    EXPERIENCE_DISMISSED("io.rover.ExperienceDismissed", "Experience Dismissed"),
+    EXPERIENCE_VIEWED("io.rover.ExperienceViewed", "Experience Viewed"),
+    SCREEN_PRESENTED("io.rover.ScreenPresented", "Screen Presented"),
+    SCREEN_DISMISSED("io.rover.ScreenDismissed", "Screen Dismissed"),
+    SCREEN_VIEWED("io.rover.ScreenViewed","Screen Viewed"),
+    BLOCK_TAPPED("io.rover.BlockTapped", "Block Tapped");
+}
```

Comment:
Hey @samwoodall sorry to foist this on you last minute, but I'm realizing that the English Names should go into EventAnalyticsService.  You'll have to break this enum up into two separate concerns to do so, sadly.